CHANGES_NEXT_RELEASE

```diff
@@ -11,7 +11,8 @@
 - Fix: NGSIv1 updateContext APPEND on existing geo:point attributes (and probably other geo: attributes) failing to update location (#2770)
 - Fix: bug when tenants are over 44 bytes long (#2811)
 - Add: Support for null values in string filters (#2359)
-- Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280)
+- Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
+- Fix: empty entity ids not accepted at subscription creation (#2184). [ This was fixed around June 2016 ]
```

Comment:
Looking to that commit info (in the "header" of the web page) we see that the first tag including that commit was 1.3.0 so that is probably the release which changelog needs to be adjusted.